speaking-portal-project/src/types.ts

```diff
@@ -1,52 +1,129 @@
 export interface AudioFile {
     soundFile: string
-    audioDuraion: number
+    audioDuration: number
 }
 
 export type MouthCue = {
     start: number
     end: number
-    value: string
+    value: Phoneme
 }
 
 export type MouthCueArray = Array<MouthCue>
 
+export type Phoneme = 'A' | 'B' | 'C' | 'D' | 'E' | 'F' | 'G' | 'H' | 'X'
+
+// TODO: update when more avatars added
+export enum AvatarNames {
+    barb,
+    boy,
+    __LENGTH
+}
+
+// TODO: update when more poses added
+enum Pose {
+    default,
+    relaxed,
+    __LENGTH
+}
+
+// Eyes Class for updating during runtime
+class Eyes {
+    areClosed : boolean
+    lastBlink: number
+    constructor(areClosed: boolean = false, lastBlink: number = -1) {
+        this.areClosed = areClosed
+        this.lastBlink = lastBlink
+    }
+    close(lastBlink: number) {
+        this.areClosed = true
+        this.lastBlink = lastBlink
+    }
+    open(){
+        this.areClosed = false
+    }
+}
+
+class Mouth {
+    expression: string
```

Comment:
In case we ask the artist to add expressions (happy, focused, deer-in-headlights look), then we can implement it here and do stuff with it

Reply:
great!

Reply:
Noice 👍